types/time.go

```diff
@@ -556,6 +557,50 @@ func GetFsp(s string) int8 {
 	return int8(fsp)
 }
 
+// IstTimeZone finds if there is a timezone in the string.
+// for example '2020-07-07 17:55:00.163721+08:00'.
+func IstTimeZone(s string) (tz bool) {
+	timezoneRegexp := regexp.MustCompile(`.{11,}([-|+][\d]{2}:[\d]{2})$`)
```

Comment:
Regex is much expensive in this situation, could you just use strings.LastIndex to find the last `+` or `-`?